File: PYTHON/Kode/Shirt.py
from Clothing import Clothing

class Shirt(Clothing):
    def __init__(self, brand, size, price, material):
        super().__init__(brand, size, price)
        self.material = material
    
    def get_material(self): 
        return self.material
    
    def set_material(self, material): 
        self.material = material
    
    def display(self):
        return super().display() + f" {self.material:<13} |"

File: C++/Kode/Shirt.cpp
#ifndef SHIRT_H //agar tidak terjadi tabrakan atau bahkan error, soalnya dari tadi error terus
#define SHIRT_H

#include "Clothing.cpp"

class Shirt : public Clothing {
private:
    string bahan;

public:
    Shirt(string brand, string size, double price, string bahan) : Clothing(brand, size, price) {
        this->bahan = bahan;
    }
    
    void setbahan(string bahan) { 
        this->bahan = bahan; 
    }

    string getbahan() { 
        return bahan; 
    }
    
    void display() override {
        Clothing::display();
        cout << " | " << setw(15) << bahan << " |" << endl;
    }
};

#endif